packages/marketplace/contracts/libraries/LibOrder.sol

```diff
@@ -59,10 +59,12 @@ library LibOrder {
 
     /// @notice validates order time
     /// @param order whose time we want to validate
+    // solhint-disable not-rely-on-time
     function validateOrderTime(Order memory order) internal view {
-        // solhint-disable-next-line not-rely-on-time
+        //slither-disable-next-line timestamp
         require(order.start == 0 || order.start < block.timestamp, "Order start validation failed");
-        // solhint-disable-next-line not-rely-on-time
+        //slither-disable-next-line timestamp
         require(order.end == 0 || order.end > block.timestamp, "Order end validation failed");
     }
+    // solhint-enable not-rely-on-time
```

Comment:
why not do the same for slither comments than solhint (start / end instead of the 2 comments)